be/src/exprs/block_bloom_filter.hpp

```diff
@@ -78,6 +78,13 @@ class BlockBloomFilter {
         }
     }
 
+    // This function is only to be used if the be_exec_version may be less than 2. If updated, please delete it.
+    void insert_new_hash(const Slice& key) noexcept {
```

Comment:
do not use newxxxx, just use name like insert_crc32_hash， it will be more clear

Reply:
ok,I have renamed it.